### DISCUSSION TITLE
Omni Channel Capacity Manager no longer works for agents

### DISCUSSION THREAD

**Discussion by stjohnson44** [twilio-professional-services/flex-project-template#669] [Q&A] [answered]
In debugging current customer's console logs, we have found that the Omni Channel Capacity Manager consistently fails for regular agents. This is due to the serverless function being called, which checks the user's roles before executing the channel update. Is this behavior intended, or an oversight from a previous update?

**Comment by dremin** [comment, marked as answer]
Hi @stjohnson44! This is absolutely an oversight, thanks for bringing it to our attention! I'll get this fixed.

**Reply by dremin** [reply]
This is fixed by #670.
